packages/format-library/src/link/modal-screens/link-settings-screen.native.js

```diff
@@ -42,6 +42,9 @@ const LinkSettingsScreen = ( {
 	const [ opensInNewWindow, setOpensInNewWindows ] = useState(
 		activeAttributes.target === '_blank'
 	);
+	const [ markedAsNoFollow, setMarkedAsNoFollow ] = useState(
+		( activeAttributes.rel = 'nofollow' )
```

Comment:
```suggestion
		( activeAttributes.rel?.includes( 'nofollow' ) )
```

Are there other options for this we need to consider?

Reply:
I'd say let's keep the PR's simple and small, we can always add more later but that prevents them from getting to complex or going stale

Reply:
No sorry I was thinking if the attributes have other settings on `rel`. For example `noopener` like this PR accounts for elsewhere.

Reply:
+1 to what @getdave asks as it is now it doesn't work because it expects a boolean value for `markedAsNoFollow`